### PR TITLE
DBZ-5234 Build Oracle connector by default without Maven profiles

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-5234

### CODE REVIEW

File: jenkins-jobs/pipelines/release-pipeline.groovy [hunk 1]
```diff
@@ -427,13 +427,13 @@ node('Slave') {
         stage('Prepare release') {
             dir(DEBEZIUM_DIR) {
                 sh "git checkout -b $CANDIDATE_BRANCH"
-                sh "mvn clean install -DskipTests -DskipITs -Poracle"
+                sh "mvn clean install -DskipTests -DskipITs -Poracle-xstream"
```
Comment (jpechane) [inline jenkins-jobs/pipelines/release-pipeline.groovy:430]: @Naros Are here missing commas?

Comment (Naros) [reply jenkins-jobs/pipelines/release-pipeline.groovy:430]: Nope, the new `oracle-stream` profile is required to build and include the Xstream bits as they're excluded by default.

Comment (jpechane) [reply jenkins-jobs/pipelines/release-pipeline.groovy:430]: Sorry, I misread dash as minus to disable `xstream` profile

**Comment by jpechane** [comment]
@Naros Nice one, applied!

**Pull request merged**